src/utils/Factory.ts

```diff
@@ -0,0 +1,111 @@
+/// <reference path='../app.ts' />
+
+module n3Charts.Utils {
```

Comment:
Same here, these should be (it's open to discussion, of course) in separated files, named accordingly.